docs/build-speed.md

```diff
@@ -72,47 +72,8 @@ We suggest to use [**ccache**](https://ccache.dev/) to cache the compilation of
 Ccache works by wrapping the C++ compilers, storing the compilation results, and skipping the compilation
 if an intermediate compilation result was originally stored.
 
-To install it, you can follow the [official installation instructions](https://github.com/ccache/ccache/blob/master/doc/INSTALL.md).
-
-On macOS, we can install ccache with `brew install ccache`.
-Once installed you can configure it as follows to cache NDK compile results:
-
-```
-ln -s $(which ccache) /usr/local/bin/gcc
-ln -s $(which ccache) /usr/local/bin/g++
-ln -s $(which ccache) /usr/local/bin/cc
-ln -s $(which ccache) /usr/local/bin/c++
-ln -s $(which ccache) /usr/local/bin/clang
-ln -s $(which ccache) /usr/local/bin/clang++
-```
```

Comment:
Those instructions are needed for Android, why are you removing them?

Reply:
Yes we would have to verify if it's still needed or not

Reply:
Hmm so I can't even get ccache to work on android.
From my testing the ndk compiler gets used 
/home/johnf/opt/Android/SDK/ndk/26.1.10909125/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++